ui/pages/confirm-import-token/confirm-import-token.test.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import reactRouterDom from 'react-router-dom';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { fireEvent, screen } from '@testing-library/react';
+import {
+  ASSET_ROUTE,
+  IMPORT_TOKEN_ROUTE,
+} from '../../helpers/constants/routes';
+import { addTokens, clearPendingTokens } from '../../store/actions';
+import { renderWithProvider } from '../../../test/jest/rendering';
```

Comment:
[nit] You can drop `/rendering` from this path if you like. `test/jest` exports helpers from `test/jest/rendering` as well as `screen` and `fireEvent` from RTL.

Reply:
sweeet. Thank you!